articles/evangelist-and-chameleon.md

```diff
@@ -0,0 +1,61 @@
+
+# Possibility 2: The Evangelist
+
+Whoever wants to listen (and everyone else, too) will hear continuously about all the advantages of Andrea's favourite tool. She does everything she can to have as many people as possible switch to her favourite tool, even if there are good reasons why up to now, something else was in use.
+
+The big challenge of evangelists is that they usually don't know the alternatives. This means that Andrea sounds more like "I don't want to learn about this tool" than like "I know something better". In the worst case, the functionality she points out as the biggest advantage has been implemented in an improved way in the very alternative she wants to replace.
+
+The advantage of the evangelist is that many migrations need the tenacity of an evangelist. Without evangelists, we would still use the tools from the last century.
+
+The disadvantage is that Andrea might overlook a far more important issue or a precondition for the migration.
+
+# Possibility 3: The Tightrope Walker
+
+Where possible, Andrea uses her preferred tool. Whether she adheres to the team's conventions or not is a balancing act:
+
+ * If your environment differs too much from everybody else's, pair programming will become more difficult.
+ * Most companies have rules about what can be installed/used. Because Andrea most likely doesn't know the reasoning for each of these rules, she has to consider carefully which rules she breaks.
+ * A tool you know well and that is configured to fit your preferences can increase your productivity and reduce your frustration.
+
+The difference between a tightrope walker and an evangelist is that the tightrope walker focuses on her own productivity and comfort. She doesn't attempt to convince anybody, but simply uses whatever she prefers (as long as she believes that it is ok within the context).
+
+The advantage of the tightrope walker is that because the specific tools are less important, Andrea can look at the big picture and focus on the most important issues.
+
+The disadvantage is that Andrea might miss an opportunity to improve the situation of the whole team.
+
+# Possibility 4: The Chameleon
+
+Even though Andrea has her preferred tools, she can work with many other tools. She adapts her way of working to fit the tool in use and continues to learn about each tool she uses. Given her experience with various tools, she can choose the tool that fits best to the problem at hand, regardless of her personal preferences.
+
+Like the whiner, the chameleon does not attempt to change anything. But the motivation is completely different: For the chameleon, the choice of the tool far less important than many other things. The chameleon neither attempts to convince anybody nor takes any action to change the situation. This is not due to resignation, but because the chameleon adapts and is happy with many tools.
```

Comment:
Satz fehlt ein Wort.

" For the chameleon, the choice of the tool **is** far less important than many other things."